Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```
@@ -865,6 +865,24 @@
     maxIntensity: 10
     canCreateVacuum: false
 
+- type: entity
+  id: BulletHandmadeGrenadeBlast
+  name: blast grenade
+  parent: BaseBulletTrigger
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/Projectiles/projectiles2.rsi
+    layers:
+      - state: grenade
+  - type: ExplodeOnTrigger
+  - type: Explosive
+    explosionType: Default
+    totalIntensity: 50  
+    intensitySlope: 1
+    maxIntensity: 3
```

```suggestion
    totalIntensity: 25  
    intensitySlope: 5
    maxIntensity: 3
```
This would be firebomb parity, what you have it at currently is blast grenade parity.